openlibrary/macros/databarAuthor.html

```diff
@@ -6,21 +6,21 @@
         <!-- IF ON LIST -->
         <div class="dropit">
             <div class="icon listed"></div>
-            <p class="listed">$page.get('name', 'name missing') is on your <a href="#">List Name Which Could Be Long</a> list. <span class="smaller"><a href="#" class="red">Remove</a>?</span></p>
+            <p class="listed">$page.get('name', 'name missing') is on your <a href="#">$_("List Name Which Could Be Long")</a> list. <span class="smaller"><a href="#" class="red">$_("Remove")</a>?</span></p>
```

Comment:
I believe "name missing" is a user visible string here. Also, there are several pieces of the message not wrapped:
* is on your
* list.
* the question mark (?) after "Remove" should be included since punctuation changes with language